docs/en/defined-terms.rst

```diff
@@ -23,71 +23,106 @@ Below are the description of acronyms and definitions which are useful for furth
      - Description
      - Notes
    * - User
```

Comment:
We have currently kept the "User" definition of the ARF. We need to discuss whether to use the one from the Implementing Act (‘wallet user’ means a user who is in control of the wallet unit) and whether tp keep the "Holder" term as well.

Reply:
I would keep both